stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/channel/internal/ChannelRepository.kt

```diff
@@ -160,4 +162,29 @@ internal class ChannelRepositoryImpl(
     override fun clearChannelCache() {
         channelCache.evictAll()
     }
+
+    override suspend fun updateLastMessageForChannel(cid: String, lastMessage: Message) {
```

Comment:
Maybe some docs? IDK if we have them in the interface (if not, then add them there)

Reply:
I'm adding Kdocs in this PR: https://github.com/GetStream/stream-chat-android/pull/3358